src/org/rust/lang/RustSdkType.kt

```diff
@@ -67,10 +85,22 @@ class RustSdkType : SdkType("Rust SDK") {
         }
 
         try {
-            val cmd = GeneralCommandLine()
+            val cmd = if (isMultiRust(sdkHome)) {
+                val sdkLibraryPath = getSdkLibrary(sdkHome).absolutePath
+                GeneralCommandLine()
+                    .withWorkDirectory(sdkHome)
+                    .withEnvironment(hashMapOf(
+                        "DYLD_LIBRARY_PATH" to sdkLibraryPath,
+                        "LD_LIBRARY_PATH" to sdkLibraryPath
+                    ))
+                    .withExePath(rustc.absolutePath)
+                    .withParameters("-C rpath", "--version")
```

Comment:
Yes, we need to execute cargo in mutirust slighly different way. Best way to handle it is start to use cargo and test everything on different systems.